core/util/cursor.js

```diff
@@ -80,6 +80,7 @@ export default class Cursor {
     }
 
     change(rgba, hotx, hoty, w, h) {
+        if (!this._target) { return; }
```

Comment:
Hmm... What's the corner case that needed this? And won't it result in the cursor not being properly updated?

Reply:
Hmm... But if we're not connected then there is no `RFB` object. Or did you manage to find a window whilst it was connecting?

Reply:
I am currently in a meeting, will recheck later.

Reply:
OK, you seem to be right, this is an artifact from manual console-based testing.